main.go

```diff
@@ -284,12 +297,35 @@ func WriteCRDDocs(crd *apiextensionsv1beta1.CustomResourceDefinition, outputFold
 }
 
 func main() {
+	var err error
+
+	var crdDocsGenerator CRDDocsGenerator
+	{
+		c := &cobra.Command{
+			Use:          "crd-docs-generator",
+			Short:        "crd-docs-generator is a command line tool for generating markdown files that document Giant Swarm's custom resources",
+			SilenceUsage: true,
+			RunE: func(cmd *cobra.Command, args []string) error {
+				return generateCrdDocs(crdDocsGenerator.apiExtensionsTag)
+			},
+		}
+
+		c.PersistentFlags().StringVar(&crdDocsGenerator.apiExtensionsTag, "apiExtensionsTag", "master", "The tag to use from the apiextensions repo when building crd documentation.")
```

Comment:
```suggestion
		c.PersistentFlags().StringVar(&crdDocsGenerator.apiExtensionsTag, "apiextensions-version-tef", "master", "The git version reference (tag, branch, commit SHA) to use from the giantswarm/apiextensions repository")
```

1. Command line flags should be lowercase and use `-` as separator.
2. `master` is not a tag, and I'm assuming the flag will support all sorts of git version references.

Reply:
ok! i fixed a typo, and I think technically the thing we're referring to is a commit ref, not a version ref. Or maybe even just 'ref'

Reply:
Fine with me